### DISCUSSION TITLE
Add "move" option under the 3 dots menu for editors

### DISCUSSION THREAD

**Discussion by sjwant** [outline/outline#7417] [Ideas]
Editors can already move docs via drag and drop in the sidebar, so giving them the ability to move in other ways would be nice! Especially if the sidebar is a bit cluttered

**Comment by tommoor** [comment]
<img width="289" alt="image" src="https://github.com/user-attachments/assets/a27b4a49-b0ee-4a72-af8f-50758f0c2d0e">

It's already there

**Reply by sjwant** [reply]
Hm, heres what one user is seeing. Maybe an issue on our end? 
![image (1)](https://github.com/user-attachments/assets/184f5a8e-5ef5-457c-9695-812b1aae77b2)

**Reply by tommoor** [reply]
That looks like a collection menu rather than a document menu

**Reply by sjwant** [reply]
oh that makes more sense as to why she was confused. ok close me out!
